src/payment/strategies/bolt/bolt-payment-strategy.ts

```diff
@@ -1,38 +1,142 @@
 import { isNonceLike } from '../..';
 import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
-import { MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+import { MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
 import { OrderActionCreator, OrderRequestBody } from '../../../order';
 import { OrderFinalizationNotRequiredError } from '../../../order/errors';
-import { PaymentArgumentInvalidError } from '../../errors';
+import { StoreCreditActionCreator } from '../../../store-credit';
+import { PaymentArgumentInvalidError, PaymentMethodCancelledError } from '../../errors';
+import { NonceInstrument } from '../../payment';
 import PaymentActionCreator from '../../payment-action-creator';
-import { PaymentRequestOptions } from '../../payment-request-options';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
 import PaymentStrategy from '../payment-strategy';
 
+import { BoltCheckout, BoltTransaction } from './bolt';
+import BoltScriptLoader from './bolt-script-loader';
+
 export default class BoltPaymentStrategy implements PaymentStrategy {
+    private _boltClient?: BoltCheckout;
+    private _methodId!: string;
```

Comment:
Do you really need to keep this prop? 🤔 You can always get it from the `OrderRequestBody`.